spec/services/activitypub/process_status_update_service_spec.rb

```diff
@@ -26,13 +26,14 @@
 
   let(:alice) { Fabricate(:account) }
   let(:bob) { Fabricate(:account) }
+  let(:eve) { Fabricate(:account) }
```

Comment:
Is this used anywhere?

Reply:
Apparently not. I'm surprised this didn't get caught by any of our linters.

Reply:
Me too. Pretty sure rubocop yelled at me before because of a similar situation :man_shrugging: